src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,148 @@
+        System.out.println("WEEWOOWEEWOO WELCOME! I'm Siren");
+        System.out.println("What can I do for you?");
+        horizontalLine();
+    }
+
+    /**
+     * Prints the string that is given.
+     *
+     * @param string the string to be printed out.
+     */
+    public static void print(String string) {
+        System.out.println(string);
+    }
+
+    /**
+     * Prints out a horizontal line.
+     */
+    public static void horizontalLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * Prints the goodbye message when the user exits the program.
+     */
+    public static void goodbyeMessage() {
+        System.out.println("WEEEWOOWEEWOO GOODBYE! Hope to see you again soon!");
+    }
+
+    /**
+     * Prints the exception's message.
+     *
+     * @param e the exception that will have its message printed.
+     */
+    public static void showExceptionError(DukeException e) {
+        System.out.println(e.getMessage());
+    }
+
+    /**
+     * Prints the message to indicate a saved file is found.
+     */
+    public static void savedFileFound() {
+        horizontalLine();
+        System.out.println("BLINGBLING! You have a saved file! Displaying the contents to you (if any)!");
+    }
+
+    /**
+     * Prints the message to indicate there is no saved file found.
+     */
+    public static void savedFileNotFound() {
+        horizontalLine();
+        System.out.println("OH NO! I couldn't find a \"duke.txt\" file in your data directory, "
+                + "I'll be creating one for you!");
+        horizontalLine();
+    }
+
+    /**
+     * Prints the message to indicate the specific directory cannot be found.
+     */
+    public static void directoryNotFound() {
+        horizontalLine();
+        System.out.println("OH NO! I couldn't find a \"data\" directory in your project root directory, "
+                + "I'll be creating one for you!");
+    }
+
+    /**
+     * Prints the message when the user deletes a task.
+     *
+     * @param description the description of task that was deleted.
+     * @param arraySize the number of tasks left.
+     */
+    public static void deleteTaskOutput(String description, int arraySize) {
+        System.out.println("ALRIGHTY! I've removed this task:");
+        System.out.println(description);
+        System.out.println("Now you have " + arraySize + " tasks in the list.");
+    }
+
+    /**
+     * Prints the message when the user adds a task.
+     *
+     * @param taskList the TaskList object that contains the array list containing the tasks.
+     */
+    public static void addedTaskOutput(TaskList taskList) {
+        System.out.println("DINGDONG GOT IT! I've added this task:");
+        System.out.println(taskList.taskArray.get(taskList.taskArray.size() - 1));
+        System.out.println("Now you have " + taskList.taskArray.size() + " tasks in the list.");
+    }
+
```

Comment:
Some inconsistent tense for method names here could perhaps prove confusing. Maybe try to standardise it to present tense or the imperative mood? Like `deleteTaskOutput`, `addTaskOutput`, `markTask`, etc.